### ISSUE TITLE
losses

### ISSUE THREAD

**Issue by PardisTaghavi** [compphoto/SIDepth#1]
Thanks for the great work.
Are you planing to release losses used on the paper?

**Comment by miangoleh** [comment]
Hi. I am not sure when I will be able to share the training code but I definitely can share the loss functions quickly. Which loss function do you need?

**Comment by PardisTaghavi** [comment]
thanks for the response. I need loss function for "high-resolution ssi depth estimation" section on the paper. It says Lssi, Lso and Lssig is used for that network.

**Comment by YongtaoGE** [comment]
Hi, thanks for your great work.  Will you update the loss functions proposed in the paper in this repo?

**Comment by miangoleh** [comment]
@YongtaoGe I will not be able to include the loss functions in the repo yet. I will release the full training code later. However if you need specific loss functions I can share those with you.

**Comment by YongtaoGE** [comment]
@miangoleh, it is appreciated if you could share with me the Sparse ordinal loss, and loss functions mentioned in Eq 7 in the paper. My email address is yongtaoo.ge@gmail.com. Thanks

**Comment by Leoshiou** [comment]
@miangoleh Hi, thanks for the great work. Is it possible to share the implementation of the loss functions mentioned in the paper? Thank you very much.
